src/components/shared/error-messages.tsx

```diff
@@ -37,6 +37,9 @@ export const ERROR_MESSAGES = {
 	EMAIL: {
 		INVALID: "Invalid email address",
 	},
+	ESIGNATURE: {
+		UPLOAD: "Signature not uploaded.",
```

Comment:
the `.` shows for `ESIGNATURE.UPLOAD` but not for `COMMON.FIELD_REQUIRED`, would need to standardise?

Reply:
will resolve it in another PR because a couple of error messages cannot be overridden. 
will update components to allow overriding the messages through the validation config